R/split_funs.R

```diff
@@ -79,6 +79,9 @@ setGeneric(".applysplit_ref_vals",
 #' functions and modify either the incoming data before they are called or
 #' their outputs.
 #'
+#' @seealso [make_split_fun()] for a simplified interface for creating custom split 
```

Comment:
please change "a simplified interface" to "the API", as in "... for the API for creating custom split functions"